### PR TITLE
Fuzzers: Add a WebP fuzzer

### PR DESCRIPTION
**Pull request by nico**

### CODE REVIEW

File: Meta/Lagom/Fuzzers/FuzzWebPLoader.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020, the SerenityOS developers.
```
Comment (skyrising) [inline Meta/Lagom/Fuzzers/FuzzWebPLoader.cpp:2]:
```suggestion
 * Copyright (c) 2023, the SerenityOS developers.
```

Comment (timschumi) [inline Meta/Lagom/Fuzzers/FuzzWebPLoader.cpp:2]: 2020?

File: Meta/Lagom/Fuzzers/CMakeLists.txt [hunk 2]
```diff
@@ -75,6 +75,7 @@ add_simple_fuzzer(FuzzUTF16BEDecoder LibTextCodec)
 add_simple_fuzzer(FuzzRSAKeyParsing LibCrypto)
 add_simple_fuzzer(FuzzVP9Decoder LibVideo)
 add_simple_fuzzer(FuzzWAVLoader LibAudio)
+add_simple_fuzzer(FuzzWebPLoader LibGfx)
```
Comment (linusg) [inline Meta/Lagom/Fuzzers/CMakeLists.txt:78]: Wasm < WebP :^)

Comment (nico) [reply Meta/Lagom/Fuzzers/CMakeLists.txt:78]: Done.

In ASCII order, WOFF < Wasm too – want me to move that up? What's our case folding policy for sorting cmake files? (`:sort` in vim would sort it up.)

Comment (linusg) [reply Meta/Lagom/Fuzzers/CMakeLists.txt]: No strong feelings, I think more files use case-insensitive ordering but it's definitely not fully consistent.

Comment (nico) [reply Meta/Lagom/Fuzzers/CMakeLists.txt]: Let's keep that as-is for now then :)

**Pull request merged**